app/src/main/java/com/github/se/icebreakrr/model/profile/ProfilesRepositoryFirestore.kt

```diff
@@ -177,17 +176,17 @@ class ProfilesRepositoryFirestore(
         }
   }
 
-  override fun getBlockedProfiles(
-      blockedProfiles: List<String>,
+  override fun getMultipleProfiles(
+      uidList: List<String>,
       onSuccess: (List<Profile>) -> Unit,
       onFailure: (Exception) -> Unit
   ) {
-    if (blockedProfiles.isEmpty()) {
+    if (uidList.isEmpty()) {
       onSuccess(emptyList())
       return
     }
     db.collection(collectionPath)
-        .whereIn("uid", blockedProfiles)
+        .whereIn("uid", uidList)
```

Comment:
`uid` can be in a const variable

Reply:
True, I haven't coded this function but i can set this string as a constant